Annotato/Annotato/Views/Document/Edit/Annotation/AnnotationMarkdownView.swift

```diff
@@ -75,10 +96,35 @@ extension AnnotationMarkdownView: UITextViewDelegate {
             return
         }
         editView.resizeFrameToFitContent()
-        self.frame.size = editView.frame.size
-        self.heightConstraint.constant = self.frame.height
         viewModel.setContent(to: text)
-        viewModel.setHeight(to: frame.height)
+        changeSize(to: editView.frame.size)
+    }
+}
+
+extension AnnotationMarkdownView: WKNavigationDelegate {
+    // References: https://stackoverflow.com/questions/27850792/uiwebview-dynamic-content-size
+    func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
+        webView.frame.size = CGSize(width: frame.width, height: webView.scrollView.contentSize.height)
```

Comment:
When I print `webView.frame.size` after this line, I always get (some number, 0). I don't know why the webview scroll height is 0 after it has finished loading the content.

Reply:
what's the issue? are you investigating problem no. 2?

Reply:
yeah. when this method is called after the web view has finished loading the content. It always prints zero for the first time i click the eye icon. Subsequent clicks for the eye icon results in the correct height.